nre/validator_tasks.md

```diff
@@ -45,25 +45,17 @@ To run an Iota Validator a machine with the following is required:
 
 ## Deployment
 
-Iota Node can be deployed in a number of ways.
+IOTA Node can be deployed in a number of ways.
 
 There are pre-built container images available in [Docker Hub](https://hub.docker.com/r/iotaledger/iota-node/tags).
 
-And pre built `linux/amd64` binaries available in S3 that can be fetched using one of the following methods:
+And pre-built `linux/amd64` binaries available in [Github](https://github.com/iotaledger/iota/releases):
 
-```shell
-wget https://releases.iota.io/$IOTA_SHA/iota-node
-```
-
-```shell
-curl https://releases.iota.io/$IOTA_SHA/iota-node -o iota-node
-```
-
-To build directly from source:
+To build directly from source (select the branch you want to build from depending on the network you want to connect to):
 
 ```shell
 git clone https://github.com/iotaledger/iota.git && cd iota
-git checkout [SHA|BRANCH|TAG]
+git checkout [devnet|testnet|mainnet]
```

Comment:
Not sure about this. We don't have those branches. Is it planned?

Reply:
devnet and testnet already exists, mainnet will be created before the mainnet launch

Reply:
Ah, missed that, 👍